assets/src/scss/editorOverrides.scss

```diff
@@ -126,6 +126,14 @@
   }
 }
 
+.wp-block-button.transparent-button .wp-block-button__link[role="textbox"] {
```

Comment:
I haven't added the `:hover` styles here because it looked weird when editing the button